flink-table/flink-sql-parser/src/main/codegen/templates/Parser.jj

```diff
@@ -8001,6 +8003,8 @@ SqlPostfixOperator PostfixRowOperator() :
 |   < DATETIME_INTERVAL_PRECISION: "DATETIME_INTERVAL_PRECISION" >
 |   < DATETIME_TRUNC: "DATETIME_TRUNC" >
 |   < DAY: "DAY" >
+|   < DAYOFWEEK: "DAYOFWEEK" >
```

Comment:
update documentation, do we also support those in runtime code? or at least throw a proper error message?